src/SWP/Bundle/CoreBundle/Model/FacebookInstantArticlesFeedInterface.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\FacebookInstantArticlesBundle\Model\PageInterface;
+use SWP\Component\Common\Model\TimestampableInterface;
+
+interface FacebookInstantArticlesFeedInterface extends TimestampableInterface
```

Comment:
you should keep here only `public function getContentBucket();`, create interface (with the rest methods signatures) inside FacebookInstantArticles component. Then this interface would extend from the one in the fbia component